source/features/markdown.js

```diff
@@ -0,0 +1,45 @@
+import {h} from 'dom-chef';
+
+function styleInlineCode(md) {
+	return (
+		<code class="refined_twitter_markdown">
+			{md}
+		</code>
+	);
+}
+
+function isElement(el) {
+	return el instanceof HTMLElement;
+}
+
+function splitTextReducer(frag, text, index) {
+	if (index % 2) { // Code is always in odd positions
+		frag.append(styleInlineCode(text));
+	} else if (text.length > 0) {
+		frag.append(text);
+	}
+
+	return frag;
+}
+
+export default function () {
+	const splittingRegex = /`(.*?)`/g;
```

Comment:
Why isn't this regex ``/`(.+)`/g``?

I don't see the need to replace 'empty' backticks with a inline code block.

Reply:
My regex is bad ... Will fix!